dashboard/client/src/components/ActorTable.tsx

```diff
@@ -327,7 +327,7 @@ const ActorTable = ({
 
   return (
     <React.Fragment>
-      <div style={{ flex: 1, display: "flex", alignItems: "center" }}>
+      <div style={{ flex: 1, display: "flex", alignItems: "end" }}>
```

Comment:
Is this required for the migration or just a style change? Are there other non-required changes in the PR? Would prefer to keep this PR just scoped to the necessary changes for the migration, to make it easier to review the style changes separately

Reply:
This is necessary to make things look good after the migration. There seems to be slight layout issues with the new UI components.

Reply:
Gotcha, thanks